Tests.Unit.Core.ApplicationServices/ApplicationServices/Organizations/OrganizationUnitServiceTest.cs

```diff
@@ -241,6 +241,69 @@ public void TransferSelectedOrganizationRegistrations_Returns_Forbidden(bool isU
             Assert.Equal(OperationFailure.Forbidden, result.Value.FailureType);
         }
 
+        [Fact]
+        public void GetAccessRights_Returns_NotFound_When_Organization_Was_NotFound()
+        {
+            var unitUuid = A<Guid>();
+            var orgUuid = A<Guid>();
+
+            ExpectGetOrganizationReturns(orgUuid, new OperationError(OperationFailure.NotFound));
+
+            var result = _sut.GetAccessRights(orgUuid, unitUuid);
+
+            Assert.True(result.Failed);
+            Assert.Equal(OperationFailure.NotFound, result.Error.FailureType);
+        }
+
+        [Fact]
+        public void GetAccessRights_Returns_NotFound_When_OrganizationUnit_Was_NotFound()
+        {
+            var unitUuid = A<Guid>();
+            var orgUuid = A<Guid>();
+
+            var organization = new Organization() {Uuid = orgUuid};
+
+            ExpectGetOrganizationReturns(orgUuid, organization);
+
+            var result = _sut.GetAccessRights(orgUuid, unitUuid);
+
+            Assert.True(result.Failed);
+            Assert.Equal(OperationFailure.NotFound, result.Error.FailureType);
+        }
+
+        [Fact]
+        public void GetAccessRights_Returns_AccessRights()
+        {
+            var unitUuid = A<Guid>();
+            var orgUuid = A<Guid>();
+
+            var unit = new OrganizationUnit
+            {
+                Uuid = unitUuid,
+                Parent = new OrganizationUnit(),
+                Origin = OrganizationUnitOrigin.Kitos
+            };
+            var organization = new Organization() {Uuid = orgUuid, OrgUnits = new List<OrganizationUnit> {unit}};
```

Comment:
Please remember Arrange,Act,assert comments in your tests. .. this improves readability